### ISSUE TITLE
Please add results of your model on test set

### ISSUE THREAD

**Issue by vakadanaveen** [soloist97/region-hierarchical-pytorch#1]
Hi,

I appreciate your work on creating pytorch version for this project. It would be very helpful if you could also post the results file that you got after training the model. It would easy for us to compare with other models if this file is available. Thank you so much !

**Comment by soloist97** [comment]
Sure, please check this [link](https://1drv.ms/u/s!AmN4YCVEJTAIhdg66QtBbrpr1RCjxw?e=ojUmfY) to get files and the checkpoint.

**Comment by jackning10** [comment]
Hi,
great work!
Can you please share the files again I'm unable to access it from the above link. It would be great help.
Thank you

**Comment by soloist97** [comment]
@jackning10 I have checked that the link is still accessible, maybe you can tell me your email and I can send those files to you directly.

**Comment by jackning10** [comment]
Thanks that would great help. jackling822@gmail.com

**Comment by soloist97** [comment]
@jackning10 done it.

**Comment by codefreakSubham** [comment]
Hi,
Can you please share the files to me also, I'm unable to access it from the above link. It would be very helpful.
Email id: cs21s058@smail.iitm.ac.in
Thank you

**Comment by codefreakSubham** [comment]
> Hi, Can you please share the files to me also, I'm unable to access it from the above link. It would be very helpful. Email id: [cs21s058@smail.iitm.ac.in](mailto:cs21s058@smail.iitm.ac.in) Thank you

Received...Thank you so much.

**Comment by sujiaqi-1** [comment]
> @jackning10 I have checked that the link is still accessible, maybe you can tell me your email and I can send those files to you directly.
Hi!
Great work!
 I'm also unable to access it from the above link. Could you please share the files to me, it would be very helpful for me!
This is my email: sujiaqi8866@gmail.com .
Thanks very much!

**Comment by sujiaqi-1** [comment]
> Sure, please check this [link](https://1drv.ms/u/s!AmN4YCVEJTAIhdg66QtBbrpr1RCjxw?e=ojUmfY) to get files and the checkpoint.

Hi!
Great work!
I'm also unable to access it from the above link. Could you please share the files to me, it would be very helpful for me!
This is my email: [sujiaqi8866@gmail.com](mailto:sujiaqi8866@gmail.com) .
Thanks very much!

**Comment by ZKCCZ** [comment]
unable to access it from the above link. Can you please share the files to me also?
cachoacn@gmail.com
thanks so much!!
